doc/commands/history.8.rst

```diff
@@ -0,0 +1,83 @@
+..
+    Copyright Contributors to the libdnf project.
+
+    This file is part of libdnf: https://github.com/rpm-software-management/libdnf/
+
+    Libdnf is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 2 of the License, or
+    (at your option) any later version.
+
+    Libdnf is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
+
+..
+    # TODO(jkolarik): Command not ready yet in upstream ...
+
+.. _history_command_ref-label:
+
+################
+ History Command
+################
+
+Synopsis
+========
+
+``dnf5 history <subcommand> [options] [<transaction-spec>]``
+
+
+Description
+===========
+
+The ``history`` command in ``DNF5`` allows the user to view what has happened in past transactions
+and offers several operations above these transactions, like undoing and redoing them. In order to
```

Comment:
The "above" doesn't sound right to me here.
I might be wrong but I would prefer "with" or "on".